CMSgov/mint-app
# [NOREF] Update pre-commit hooks

# No Ticket

## Changes and Description

- This is almost an exact clone of https://github.com/CMSgov/easi-app/pull/1727

## How to test this change

- Pull this branch down
- Make changes that you think should break the linting process
- Run pre-commit run --all-files

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
